package.json

```diff
@@ -30,6 +30,7 @@
     "@babel/preset-env": "^7.0.0",
     "async-retry": "^1.0.0",
     "axios": "^0.21.1",
+    "fs-extra": "^9.1.0",
```

Comment:
If this is only for empty dir, perhaps
`fs.rmdir(dir, { recursive: true })` can be used instead?

Reply:
This one does not work properly when directory has files. fs-extra removed files from the dear. I think it is good to have it in our package for extra methods related to files.